README.md

```diff
@@ -46,6 +46,11 @@ setlocal DISABLEDELAYEDEXPANSION
 c:\path\to\php.exe "c:\path\to\pickle.phar" %*
 ```
 
+On linux systems, you can create shortcut, so command will be available globally:
```

Comment:
Well, that's not a shortcut ;P
You simply need to have pickle in a directory listed in the PATH environment variable, and to be sure it's executable (`chmod +x pickle`)

Reply:
Yes, whatever it is called. Just having it available from any location with just typing `pickle` (instead of `usr/pickle` as stated in readme) will be nice to have, so, however you like, it will be good if you added in readme, a simple instruction "how to add pickle (with chmod-ed flags)" in PATH.

Reply:
There's already a note about this: see

> And finally you can add it to your path or copy it in /usr/local/bin or your favorite binary directory.

a few lines above.